app/src/main/java/org/simple/clinic/patient/businessid/Identifier.kt

```diff
@@ -40,7 +40,15 @@ data class Identifier(
       }
       BangladeshNationalId -> value
       EthiopiaMedicalRecordNumber -> value
-      IndiaNationalHealthId -> value
+      IndiaNationalHealthId -> {
+        val enteredCode = IndiaNationalHealthId.enteredCodeLength(this)
+        val prefix = enteredCode.substring(0, 2)
+        val subString1 = enteredCode.substring(2,6)
+        val subString2 = enteredCode.substring(6,10)
+        val suffix = enteredCode.substring(10)
+
+        "$prefix${Unicode.nonBreakingSpace}$subString1${Unicode.nonBreakingSpace}$subString2${Unicode.nonBreakingSpace}$suffix"
```

Comment:
Probably not for this PR, but maybe we can look into how we can extract out adding spacing to IDs into a common function or something like that. A factory perhaps.

Reply:
Yes will take a look at it